src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -133,7 +133,7 @@
     <value>Created:</value>
   </data>
   <data name="Path" xml:space="preserve">
```

Comment:
Is this being used in the properties window?

Reply:
![image](https://github.com/files-community/Files/assets/66369541/4c0437c9-c1e1-4b4a-9249-4546db8528f3)
"Location:" is used instead of "Path:". I can find no other place it is used.